frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -0,0 +1,215 @@
+import { FormSection } from 'components/common/form/styles';
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { Dictionary } from 'interfaces/Dictionary';
+import {
+  IContactAddress,
+  IContactMethod,
+  IContactOrganization,
+  IContactPerson,
+} from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { FaCircle, FaRegBuilding } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+
+export interface OrganizationViewProps {
+  organization: IContactOrganization;
+}
+
+interface ContactInfoField {
+  info: string;
+  label: string;
+}
+
+interface AddressField {
+  address: IContactAddress;
+  label: string;
+}
+
+const OrganizationView: React.FunctionComponent<OrganizationViewProps> = ({ organization }) => {
+  const phoneTypes: Dictionary<string> = {};
+  phoneTypes[ContactMethodTypes.WorkMobil] = 'Mobile';
+  phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
+  phoneTypes[ContactMethodTypes.PersPhone] = 'Home';
+  phoneTypes[ContactMethodTypes.Fax] = 'Fax';
+
+  const personPhoneNumbers: ContactInfoField[] = getContactInfo(organization, phoneTypes);
+
+  const emailTypes: Dictionary<string> = {};
+  emailTypes[ContactMethodTypes.WorkEmail] = 'Work';
+  emailTypes[ContactMethodTypes.PerseEmail] = 'Personal';
+  const personEmails: ContactInfoField[] = getContactInfo(organization, emailTypes);
+
+  let personAddresses: AddressField[];
+  if (organization.addresses === undefined) {
+    personAddresses = [];
+  } else {
+    personAddresses = organization.addresses.reduce(
```

Comment:
nit: could be more concise using organization?.addresses.reduce(... ?? [] rather then needing an if/else block.

Reply:
I think I tried this and the compiler didnt like it. I could give it a second go